projects/vdk-plugins/vdk-jobs-troubleshooting/src/vdk/plugin/jobs_troubleshoot/troubleshoot_configuration.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2021 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+from vdk.internal.core.config import ConfigurationBuilder
+
+TROUBLESHOOT_UTILITIES_TO_USE = "TROUBLESHOOT_UTILITIES_TO_USE"
+TROUBLESHOOT_PORT_TO_USE = "TROUBLESHOOT_PORT_TO_USE"
+
+
+def add_definitions(config_builder: ConfigurationBuilder):
+    """
+    Here we define what configuration settings are needed for the
+    Jobs Troubleshooting plugin with reasonable defaults.
+    """
+    config_builder.add(
+        key=TROUBLESHOOT_UTILITIES_TO_USE,
+        default_value=None,
+        description="""
+            An unordered comma-separated list of strings, indicating what
+            troubleshooting utilities are to be used. E.g., "utility1,utility2".
```

Comment:
Where can I see list of available utilities ?

Reply:
There are no utilities, yet. I was thinking of adding the list of utilities in the plugin's README. Or, do you think there should be a command that returns a list of all utilities?

Reply:
I think this description should say where t find the list of existing utilities. If it can list them automatically even better. 

It's fine that they are listed in the README for now . We will have only 1 anyhow. 

But when you list them maybe include a link here. See full list at http://xx

Reply:
Updated description. Thanks for the review